src/control/security/grpc_authorization_test.go

```diff
@@ -142,6 +142,9 @@ func TestSecurity_AllRpcsAreAuthorized(t *testing.T) {
 
 			var rpcNames []string
 			for i := 0; i < svcType.NumMethod(); i++ {
+				if strings.HasPrefix(svcType.Method(i).Name, "mustEmbedUnimplemented") {
```

Comment:
Curious to know where this comes from. I noticed the control and management services are using some protobuf unimplemented type now, and I'm guessing it has something to do with that. Would be nice to comment this with some background.

Reply:
It's now a requirement that protobuf service implementations embed this struct in order to allow new methods to be added without requiring existing implementations to implement them. I'm having trouble finding a reference to it in the documentation though, so I guess most people encounter/resolve this when they try to build. https://github.com/grpc/grpc-go/issues/3669 has some discussion and links to the original issue/motivation. tl;dr: Adding this requirement brings the Go implementation into parity with implementations in other languages.

Reply:
Okay. So I guess that means this test ignores unimplemented methods, which makes sense to me.